functions-and-operators/string-functions.md

```diff
@@ -1172,12 +1172,65 @@ Return a set of comma-separated strings that have the corresponding bit in bits
 
 ### [`MID()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_mid)
 
-Return a substring starting from the specified position.
+The `MID(str, start, end)` function returns a substring starting from the specified `start` position and ends at the optinal `end` position.
```

Comment:
Please report a bug, since it looks like the `end` position is not optional in TiDB:
```mysql
tidb> select mid("kalle", 2);
ERROR 1582 (42000): Incorrect parameter count in the call to native function 'mid'
```

Reply:
https://github.com/pingcap/tidb/issues/52420